APM/CommandSubscriberRegistry.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Bundle\MongoDBBundle\APM;
+
+use MongoDB\Driver\Monitoring\CommandSubscriber;
+use function method_exists;
+
+final class CommandSubscriberRegistry
+{
+    /** @var CommandSubscriber[] */
+    private $commandSubscribers;
+
+    public function __construct(iterable $commandSubscribers)
+    {
+        $this->commandSubscribers = $commandSubscribers;
+    }
+
+    public function register() : void
+    {
+        foreach ($this->commandSubscribers as $subscriber) {
+            if (! method_exists($subscriber, 'register')) {
+                continue;
+            }
+
+            $subscriber->register();
```

Comment:
Since [CommandSubscriber](https://www.php.net/manual/en/class.mongodb-driver-monitoring-commandsubscriber.php) has no `register()` and `unregister()` methods, I don't believe we'd ever reach this point. Subscribers are registered with the extension globally via [these functions](https://www.php.net/manual/en/ref.monitoring.functions.php).

The [APM tutorial](https://www.php.net/manual/en/mongodb.tutorial.apm.php) has some examples that tie this all together.